### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by foerster-rwth** [schrebra/Windows.10.DNS.Block.List#6] [General]
Fab work.

I use Windows 10 Education (which is basically Enterprise), and I've come to regard Windows as probably the biggest malware of them all. But at the same time it's also a fairly good OS. I have already blocked every telemetry that Windows Education lets you switch off, but that of course isn't very much of the whole snooping that is going on behind the scenes.

I was initially a bit apprehensive to add your list to my Adguard server, as you write

"The network icon will show that you have no internet connectivity, this is because of msftconnecttest.com."

After installing the list, my internet connection did indeed go down, and the network icon changed to "No internet access". But only for a minute or so. Now everything seems to be normal as before. I am wondering what this means.

Cheers!

**Comment by schrebra** [comment]
Msftconnecttest.com is a domain used by Microsoft's operating systems, specifically Windows 10, to check the connectivity status of a device to the internet. When a device connects to a network, Windows 10 attempts to access the domain msftconnecttest.com to determine if it can successfully connect to the internet.The purpose of this test is to ensure that the device has a working internet connection, and if it does not, Windows 10 will display a notification indicating that the device is not connected to the internet. This feature is useful for users who want to quickly diagnose internet connectivity issues on their devices.
